### PR TITLE
Feature/new buttom to check api again in health check

### LINKED ISSUE #2910: Manager/api status changes are not checked during health check
| Wazuh | Elastic | Rev |
| ----- | ------- | --- |
| 4.1.0  | 7.10.0     | 4101-1 |

Hello team,

When entering in wazuh app, if the manager or API is down, it is indicated in the initial health-check:

image

but if we start the wazuh manager (checking that it has been connected successfully), the health-check status does not change, 
even though an animation is displayed indicating the opposite case.

animation

Below I show the commented case with an example:

https://user-images.githubusercontent.com/23462183/106713690-1d6fe080-65fb-11eb-93ea-572b1c2b730f.mp4

It is proposed to check if the status of the manager or API has changed in which case update the health-check and continue to the wazuh app dashboard.

Best regards.

### PR DESCRIPTION
**Pull request by CPAlejandro**
Hi guys, we resolve:

- When an error occurred, the wazuh logo of the health-check kept loading, we have fixed it
- We have also created a button to be able to reconnect the api, if it was down when doing the health-check, but it is already working
Closes #2910

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.1.3 - Kibana 7.10.0 , 7.10.2 - Revision 4106
```
Comment (frankeros) [inline CHANGELOG.md:5]: move to `v4.2.0` and `Revision 4201` please

File: server/controllers/wazuh-api.ts [hunk 2]
```diff
@@ -321,7 +321,7 @@ export class WazuhApiCtrl {
         );
       }catch(error){
         return ErrorResponse(
-          `ERROR3099 - ${error.response.data.detail || 'Wazuh not ready yet'}`,
+          `ERROR3099 - ${error.response?.data?.detail || 'Wazuh not ready yet'}`,
```
Comment (frankeros) [inline server/controllers/wazuh-api.ts:324]: test it with build and Kibana prod please

File: CHANGELOG.md (hunk 1 above)
Comment (CPAlejandro) [reply CHANGELOG.md]: Done!

**Review by pablomarga — APPROVED** [review]
Tested it and everything seems to be working fine, good job! LGTM!

**Pull request merged**